SimPEG/Regularization.py

```diff
@@ -684,6 +739,9 @@ def __init__(
     cell_weights = properties.Array(
         "regularization weights applied at cell centers", dtype=float
     )
+    scale = properties.Float(
```

Comment:
where does this come in? how is it different than cell-weights or the alphas?

Reply:
Issue with alpha's is that they are only applied during the summation of the ```ObjectiveFunction``` but don't show up if you just ask for ```objcts[ ].deriv```. Could do the cell_weights, but it's getting a bit hairy with the order of operation when doing normalization. Can we just have it as a hidden switch?

Reply:
okay, if it doesn't need to be exposed to the user, then lets make it a private variable `_scale`

Reply:
and if you don't mind, could you try to make the docstring a bit more descriptive